README.md

```diff
@@ -69,6 +69,12 @@ If `Some`, this extension will return the value, and if `None` it will return a
 int value = Option.Some(100).ValueOrDefault(50); // Returns 100
 int value = Option.None<int>().ValueOrDefault(50); // Returns 50
 ```
+#### BindIfNone
+If `Some`, this extension will return the value, and if `None` it will return the Option produced by the delegate parameter.
+```csharp
+Option<int> value = Option.Some(100).BindIfNone(50); // Returns 100
```

Comment:
Doesn't the parameter need to be an option here?

Reply:
Yes it does, oops

Reply:
Fixed